README.md

```diff
@@ -1,4 +1,10 @@
-[![Travis Badge](https://travis-ci.org/sendgrid/csharp-http-client.svg?branch=master)](https://travis-ci.org/sendgrid/csharp-http-client)
+![SendGrid Logo](https://uiux.s3.amazonaws.com/2016-logos/email-logo%402x.png)
+
+[![BuildStatus](https://travis-ci.org/sendgrid/csharp-http-client.svg?branch=master)](https://travis-ci.org/sendgrid/csharp-http-client)
+[![Email Notifications Badge](https://dx.sendgrid.com/badge/csharp)](https://dx.sendgrid.com/newsletter/csharp)
+[![npm](https://img.shields.io/npm/l/express.svg)]()
```

Comment:
Please remove the NPM line, it doesn't apply here :)

Reply:
Ah, missed that line! Looks like the issue has been resolved, so I'll close this one.